### PR TITLE
Update to new refill percentages for credit billing

### PR DESCRIPTION
**Pull request by teesloane**
ticket: [JIRA 20226](https://circleci.atlassian.net/browse/CIRCLE-20226?utm_medium=referral-external&utm_source=jira-slack&utm_term=)

### CODE REVIEW

**Comment by rosaliebud** [comment]
@teesloane row 310 will also need to be updated.

**Comment by teesloane** [comment]
Updated! Please let me know if I understood correctly @rosaliebud !

File: jekyll/_cci2/faq.md [hunk 1]
```diff
@@ -294,7 +294,7 @@ An `active user` is any user who triggers the use of compute resources on non-OS
 {:.no_toc}
 
 On the Performance plan, when you reach 5% of your remaining credits, you will
```
Comment (rosaliebud) [inline jekyll/_cci2/faq.md]: 5% should be updated to 10%

File: jekyll/_cci2/faq.md [hunk 2]
```diff
@@ -294,7 +294,7 @@ An `active user` is any user who triggers the use of compute resources on non-OS
...
-be refilled 10% of your credits. For example, If your monthly package size is 25,000 credits, you will automatically be refilled 2,500 credits when you reach 1,250 remaining credits.
+be refilled 25% of your credits. For example, If your monthly package size is 25,000 credits, you will automatically be refilled 6,250 credits when you reach 1,250 remaining credits.
```
Comment (rosaliebud) [inline jekyll/_cci2/faq.md]: 1,250 remaining credits should be updated to 2,500 remaining credits

File: jekyll/_cci2/faq.md [hunk 3]
```diff
@@ -307,7 +307,7 @@ You can pay from inside the CircleCI app for monthly pricing.
 #### When do I pay?
 {:.no_toc}
 
-On the Usage plans, at the beginning of your billing cycle, you will be charged for user seats, premium support tiers and your monthly credit allocation. Any subsequent credit refills _during_ the month (such as the auto-refilling on reaching 5% of credits available) will be paid _at the time of the refill_.
```
Comment (rosaliebud) [inline jekyll/_cci2/faq.md]: reaching 5% of credits available should be updated to reaching 10% of credits available

**Pull request merged**